content/docs/tutorials/best-practices-with-ocm.md

```diff
@@ -266,11 +266,11 @@ eu.gcr.io/acme/simpleserver   0.1.0-linux-amd64   08641d64f612   About a minute
 In the next step we create a component archive and a transport archive
 
 ```shell
-$ PROVIDER=acme
-$ COMPONENT=github.com/$(PROVIDER)/simpleserver
-$ VERSION=0.1.0
-$ mkdir gen
-$ ocm create ca ${COMPONENT} ${VERSION} --provider ${PROVIDER} --file gen/ca
+PROVIDER=acme
```

Comment:
I am undecided if the `$` should remain, since they show that these are cli-commands. However, I don't know how we do that on other occasions.

Reply:
nevermind, i don't like `$` at the beginning of the line